UKHomeOffice/sgmr-service
# 2.0 SPCR-371 Edit a person

## Description


## To test

- go to /people
- click on a person
- > person's details should be populated into the fields
- change the persons name
- click continue
- click save
- > you should be returned to the people list with the person's new name
- click on a person
- click 'delete'
- > you should be returned to the people list with that person no longer there
- click on a person
- change their name
- click exit without saving
- > you should be returned to the people list without that person being changed


## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
